src/DI.Specification.Tests/DI.Specification.Tests.csproj

```diff
@@ -5,6 +5,8 @@
   <PropertyGroup>
     <Description>Suite of xUnit.net tests to check for container compatibility with Microsoft.Extensions.DependencyInjection.</Description>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>Microsoft.Extensions.DependencyInjection.Specification.Tests</AssemblyName>
+    <RootNamespace>Microsoft.Extensions.DependencyInjection.Specification</RootNamespace>
```

Comment:
Oh, this is in `src` not `test`. Still, all the namespaces in this project are `Microsoft.Extensions.DependencyInjection.Specification` not `Microsoft.Extensions.DependencyInjection.Specification.Tests`.